src/illuminatio/rule.py

```diff
@@ -45,10 +60,16 @@ def __repr__(self):
         return self.__str__()
 
     def to_dict(self) -> dict:
+        """
+        converts a rule into a dictionary
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)